packages/webapp/src/pages/Connection/List.tsx

```diff
@@ -175,66 +179,112 @@ export default function ConnectionList() {
+                                    ({
+                                        id,
+                                        connection_id: connectionId,
+                                        provider,
+                                        provider_config_key: providerConfigKey,
+                                        created: creationDate,
+                                        active_logs,
+                                        connection_config
+                                    }) => (
+                                        <div
+                                            key={`tr-${id}`}
+                                            className={`flex gap-4 ${
+                                                id !== connections.at(-1)?.id ? 'border-b border-border-gray' : ''
+                                            } min-h-[4em] px-2 justify-between items-center hover:bg-hover-gray cursor-pointer`}
+                                            onClick={() => {
+                                                navigate(`/${env}/connections/${encodeURIComponent(providerConfigKey)}/${encodeURIComponent(connectionId)}`);
+                                            }}
+                                        >
+                                            <div className="flex items-center w-1/4 gap-2 py-2 truncate">
+                                                <span className="break-words break-all truncate">
+                                                    {connection_config?.display_name ? (connection_config.display_name as string | undefined) : '-'}
+                                                </span>
+                                                {active_logs && <ErrorCircle />}
+                                            </div>
+                                            <div className="flex w-1/4">
+                                                <span className="break-words break-all truncate">
+                                                    {connection_config?.customer_domain ? (connection_config.customer_domain as string | undefined) : '-'}
+                                                </span>
+                                            </div>
+                                            <div className="flex items-center w-1/4 gap-3">
+                                                <div className="w-7">
+                                                    <IntegrationLogo provider={provider} height={7} width={7} />
+                                                </div>
+                                                <p className="break-words break-all">{providerConfigKey}</p>
+                                            </div>
+                                            <div className="flex w-1/4 items-center">
+                                                <span className="break-words break-all truncate text-gray-400 mr-2">{truncateWithEllipsis(connectionId)}</span>
```

Comment:
Can you add a tooltip? (using Tooltip.tsx not geist)
also there is a problem with arbitrary truncate, you truncate even if it could fit in the column width. You can try to make it more flexible, or use `truncate` in tailwind it's less good but more automated